cmd/device/add/add.go

```diff
@@ -79,29 +78,29 @@ func deviceHandler(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func createDevicesFromFile() error {
+func createDevicesFromFile(cmd *cobra.Command) error {
 	devices, err := update.LoadDevicesFromFile(file)
 	if err != nil {
 		return err
 	}
 
 	client := local.New(config.Conf.Clients["Metadata"].Url() + clients.ApiDeviceRoute)
 	for _, d := range devices {
-		_, err = metadata.NewDeviceClient(client).Add(context.Background(), &d)
+		_, err = metadata.NewDeviceClient(client).Add(cmd.Context(), &d)
 		if err != nil {
 			fmt.Println("Error: ", err.Error())
 		}
 	}
 	return nil
 }
 
-func parseDevice(interactiveMode bool) ([]models.Device, error) {
+func parseDevice(cmd *cobra.Command, interactiveMode bool) ([]models.Device, error) {
```

Comment:
why did you make parseDevice take a cobra.Command argument instead of just context?